provider-example/src/lib.rs

```diff
@@ -61,8 +61,8 @@ pub static TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305_SHA256: rustls::SupportedCipherS
 pub fn certificate_verifier(
     roots: rustls::RootCertStore,
 ) -> Arc<dyn rustls::client::danger::ServerCertVerifier> {
-    Arc::new(rustls::client::WebPkiServerVerifier::new_with_algorithms(
-        roots,
-        verify::ALGORITHMS,
-    ))
+    rustls::client::WebPkiServerVerifier::builder(roots.into())
```

Comment:
I think the commit message got truncated for this cmmit ("builder for server cert verifier, CRL support")

Reply:
Ooops. Thanks for noticing. Fixed.